docs/platforms/javascript/guides/nestjs/index.mdx

```diff
@@ -1,14 +1,19 @@
 ---
 title: Nest.js
 description: "Learn about using Sentry with Nest.js."
-sdk: sentry.javascript.node
-fallbackGuide: javascript.node
+sdk: sentry.javascript.nestjs
+fallbackGuide: javascript.nestjs
 categories:
   - server
 ---
 
-<PlatformContent includePath="getting-started-primer" />
-
 This guide explains how to set up Sentry in your Nest.js application.
 
+<Note>This guide is for version 8.0.0 and up of `@sentry/nestjs`.</Note>
+
+<Note>
+  This SDK is considered **experimental and in an alpha state**. It may experience breaking changes. Please reach out on
+  [GitHub](https://github.com/getsentry/sentry-javascript/issues/new/choose) if you have any feedback or concerns.
+</Note>
+
```

Comment:
```suggestion
<Note>
This guide is for version 8.0.0 and up of `@sentry/nestjs`.

This SDK is considered **experimental and in an alpha state**. It may experience breaking changes. Please reach out on
  [GitHub](https://github.com/getsentry/sentry-javascript/issues/new/choose) if you have any feedback or concerns.
</Note>

```